ecs-cli/modules/cli/configure/configure_test.go

```diff
@@ -333,3 +331,55 @@ func TestDefaultProfileDoesNotExist(t *testing.T) {
 	assert.Error(t, err, "Expected error configuring profile")
 
 }
+
+func TestMigrate(t *testing.T) {
+	configContents := `[ecs]
+cluster = defaultCluster
+aws_profile =
+region = us-west-1
+aws_access_key_id = AKID
+aws_secret_access_key = SKID
+compose-project-name-prefix =
+compose-service-name-prefix = ecs-
+cfn-stack-name-prefix =
+`
+	// Create a temprorary directory for the dummy ecs config
+	tempDirName, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Fatal("Error while creating the dummy ecs config directory")
+	}
+	os.Setenv("HOME", tempDirName)
+	defer os.Unsetenv("HOME")
+	defer os.RemoveAll(tempDirName)
+
+	// save the old config
+	fileInfo, err := os.Stat(tempDirName)
+	assert.NoError(t, err)
+	mode := fileInfo.Mode()
+	err = os.MkdirAll(tempDirName+"/.ecs", mode)
+	assert.NoError(t, err, "Could not create config directory")
+	defer os.RemoveAll(tempDirName)
+	logrus.Infof("tempdir: %s", tempDirName)
```

Comment:
is this needed?

Reply:
Yeah that log statement is not needed. I must have forgotten to remove it once I was done debugging.